components/automate-cs-oc-erchef/habitat/hooks/run

```diff
@@ -55,6 +55,13 @@ export GEM_PATH={{pkgPathFor "core/bundler"}}
 # non root svc user and we need write access to the file
 cp {{pkg.svc_config_path}}/dark_launch_features.json {{pkg.svc_data_path}}
 
+{{#if cfg.external_automate.enable}}
+# TODO(jaym): Our config should know how to deal with the token. We shouldn't be showing
```

Comment:
should we resolve this todo before merging?

Reply:
This is a fairly large chunk of work. We'd need to teach the config what's a secret and then figure out how to store it in secrets service